code/modules/projectiles/updated_projectiles/guns/shotguns.dm

```diff
@@ -401,19 +401,21 @@ can cause issues with ammo types getting mixed up during the burst.
 	var/recent_notice //world.time to see when they last got a notice.
 	var/pump_lock = FALSE //Modern shotguns normally lock after being pumped; this lock is undone by pumping or operating the slide release i.e. unloading a shell manually.
 	attachable_allowed = list(
-						/obj/item/attachable/bayonet,
-						/obj/item/attachable/reddot,
-						/obj/item/attachable/verticalgrip,
-						/obj/item/attachable/angledgrip,
-						/obj/item/attachable/gyro,
-						/obj/item/attachable/flashlight,
-						/obj/item/attachable/extended_barrel,
-						/obj/item/attachable/heavy_barrel,
-						/obj/item/attachable/compensator,
-						/obj/item/attachable/magnetic_harness,
-						/obj/item/attachable/attached_gun/flamer,
-						/obj/item/attachable/attached_gun/shotgun,
-						/obj/item/attachable/stock/shotgun)
+		/obj/item/attachable/angledgrip,
+		/obj/item/attachable/attached_gun/flamer,
+		/obj/item/attachable/bayonet,
+		/obj/item/attachable/compensator,
+		/obj/item/attachable/extended_barrel,
+		/obj/item/attachable/flashlight,
+		/obj/item/attachable/gyro,
+		/obj/item/attachable/heavy_barrel,
+		/obj/item/attachable/lasersight,
+		/obj/item/attachable/magnetic_harness,
+		/obj/item/attachable/quickfire,
```

Comment:
This list is re-ordered, which makes reviewing pain.
Please fix that so we can see more clearly what's changed and justify the added attachments.

Reply:
The list is also now in alphabetical order, which should be standard for lists. Also never in the history of anything has anyone had to justify additional attachments, but sure I'll a list of whats changed.

Reply:
The masterkey shotgun was removed.

Quickfire and laser sight was added.

Reply:
Balance changes need to be justified.
Why are you giving the shotgun QF and LS? What metas are you trying to enable, and which drawbacks are they introducing?

Reply:
quickfire does this

```	accuracy_mod = -0.20
	scatter_mod = 5
	delay_mod = -1
	burst_mod = -1
	accuracy_unwielded_mod = -0.30
	scatter_unwielded_mod = 20```

which is an even tradeoff for the shotgun considering most people get mag harness and mag harness is extremely useful using a shotgun


```	accuracy_mod = 0.15
	movement_acc_penalty_mod = -0.1
	accuracy_unwielded_mod = 0.20```

laser sight does this, which is pretty average for an underbarrel attachment

Reply:
The attachments are being currently reworked. There's no real reason to add them right now, and the meta will shift soon. Better not introduce unknown variables without a good reason.

Reply:
It's with a good reason.